lib/pdf2img.js

```diff
@@ -16,11 +16,11 @@ var options = {
 var Pdf2Img = function() {};
 
 Pdf2Img.prototype.setOptions = function(opts) {
-  options.type = opts.type || options.type;
-  options.size = opts.size || options.size;
-  options.density = opts.density || options.density;
-  options.outputdir = opts.outputdir || options.outputdir;
-  options.outputname = opts.outputname || options.outputname;
+  options.type = opts.type != null || opts.type != 'undefined' ? opts.type : options.type;
```

Comment:
This is broken in so many levels that I don't know where to start. Why have you changed these line at all? AFAIK the original implementation was correct.

Reply:
Needed to edit the lib to be used in converting list of PDF files, so far it's working on my end. Most likely my code is not clean or should have better error handling. Added a new function at the bottom that handles an array of PDF where "options" can be passed. Oh well, will need to re-visit this next time.